tools/seek_common_names.rb

```diff
@@ -57,20 +90,19 @@
   exit( 0 )
 end
 
-places_by_locale = SEEK_LOCALES.inject( {} ) do |memo,locale|
-  locale, lang, region = locale.match( /([a-z]{2})([A-Z]{2})?/ ).to_a
+places_by_locale = SEEK_LOCALES.each_with_object( {} ) do | locale, memo |
+  locale, _lang, region = locale.match( /([a-z]{2})([A-Z]{2})?/ ).to_a
   memo[locale] = if region
     Place.where( admin_level: Place::COUNTRY_LEVEL, code: region ).first
   end
-  memo
 end
 
 common_names = []
 taxon_ids = relevant_taxa.keys
-taxon_ids.in_groups_of( 1000, false ) do |group|
+taxon_ids.in_groups_of( 1000, false ) do | group |
   taxa = Taxon.where( id: group ).includes( :taxon_names )
```

Comment:
Unrelated to changes in this PR, but I noticed this could be made a little more efficient by including `place_taxon_names` as well, e.g. `.includes( { taxon_names: :place_taxon_names } )`. I can add that after merging if you don't want to make that change here

Reply:
Thanks. Added that, will merge.